packages/react-native-worklets/android/build.gradle

```diff
@@ -102,6 +102,7 @@ def IS_NEW_ARCHITECTURE_ENABLED = isNewArchitectureEnabled()
 def IS_REANIMATED_EXAMPLE_APP = safeAppExtGet("isReanimatedExampleApp", false)
 def BUNDLE_MODE = safeAppExtGet("workletsBundleMode", false)
 def WORKLETS_FEATURE_FLAGS = getStaticFeatureFlags()
+def HERMES_V1_ENABLED = safeAppExtGet("hermesV1Enabled", false)
```

Comment:
To note in case it stops working: the opt-in flag is set on the project's properties, so the `ext` entry with the flag may not exist.

Reply:
I've tested current approach and it worked 🫡